lms/views/dashboard.py

```diff
@@ -79,3 +68,28 @@ def api_assignment_stats(self):
             }
             for s in stats
         ]
+
+    def get_request_assignment(self):
```

Comment:
We don't show a nice and relevant message for both of these for now.

The goal now is to just to avoid leaking any data